docs/application/native/guides/ui/dali/actors-n.md

```diff
@@ -21,13 +23,19 @@ An actor has the following concrete types:
 - **Camera actor** determines which view of the whole virtual 3D world is rendered on the screen. By default, it is best suited for 2D applications, so you do not need to care about it in this case. For more information, see the `Dali::CameraActor` class (in [mobile](../../../../../org.tizen.native.mobile.apireference/classDali_1_1CameraActor.html) and [wearable](../../../../../org.tizen.native.wearable.apireference/classDali_1_1CameraActor.html) applications).
 - **Layer** provides a mechanism for overlaying groups of actors on top of each other. For more information, see the `Dali::Layer` class (in [mobile](../../../../../org.tizen.native.mobile.apireference/classDali_1_1Layer.html) and [wearable](../../../../../org.tizen.native.wearable.apireference/classDali_1_1Layer.html) applications).
 
-## Actors and Stage
+## Adding an Actor to the Stage
+
+Stage is a top-level object that represents the entire screen.
+It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure, which means an actor inherits a position relative to its parent, and can be moved in relation to this point.
 
-Stage is a top-level object that represents the entire screen. It is used for displaying a hierarchy of actors managed by the [scene graph](http://en.wikipedia.org/wiki/Scene_graph) structure, which means an actor inherits a position relative to its parent, and can be moved in relation to this point.
+The stage instance is a singleton object (the only instance of its class during the lifetime of the program). You can get it using a static function:
 
```

Comment:
You can get it using a static function:

Please rephrase to:

You can get a stage function using the following static function: